__tests__/__snapshots__/check-fixtures.js.snap

```diff
@@ -168,7 +227,7 @@ exports[`linting typescript.ts fixture matches snapshot 1`] = `
     "messageId": "preferExpectErrorComment",
     "nodeType": "Line",
     "ruleId": "@typescript-eslint/prefer-ts-expect-error",
-    "severity": 1,
+    "severity": 2,
```

Comment:
`@typescript-eslint/prefer-ts-expect-error` is an error in 6.0.0: https://github.com/typescript-eslint/typescript-eslint/blob/v6.0.0/packages/eslint-plugin/src/configs/strict-type-checked.ts#L65